xlsynth-g8r/src/equiv/prove_equiv.rs

```diff
@@ -88,13 +88,15 @@ pub fn ir_value_to_bv<'a, S: Solver>(
 #[derive(Debug, Clone)]
 pub struct IrFn<'a> {
     pub fn_ref: &'a ir::Fn,
+    pub pkg_ref: Option<&'a ir::Package>,
```

Comment:
Can you add a member comment on when generally pkg_ref is expected/allowed to be non-none?